torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16633,91 +16635,58 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
-    #                 dtypesIfCUDA=floating_types(),
-    #                 backward_dtypesIfCUDA=floating_types(),
-    #                 supports_inplace_autograd=False,
-    #                 decorators=[
-    #                     # Derivative wrt first tensor not implemented
-    #                     DecorateInfo(unittest.expectedFailure, "TestCommon",
-    #                                  "test_floating_inputs_are_differentiable"),
-    #                 ],
-    #                 skips=(
-    #                     # test does not work with passing lambda for op
-    #                     # AssertionError: False is not true : Tensors failed to compare as equal!
-    #                     DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
-    #                     # test fails are we permute the arguments function variant
-    #                     # but not for inplace or method.
-    #                     DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_variant_consistency_eager'),
-    #                     # TypeError: igammac(): argument 'input' (position 1) must be Tensor, not float
-    #                     DecorateInfo(unittest.skip('Skipped!'), 'TestBinaryUfuncs'),
-    #                 )),
+    BinaryUfuncInfo(
+        "igamma",
+        aliases=("torch.special.gammainc",),
+        dtypes=floating_types_and(torch.bfloat16, torch.float16),
+        dtypesIfCUDA=floating_types(),
+        lhs_make_tensor_kwargs=dict(low=0, high=10),
+        rhs_make_tensor_kwargs=dict(low=0, high=20),
+        supports_rhs_python_scalar=False,
+        supports_inplace_autograd=False,
+        supports_gradgrad=False,
+        supports_forward_ad=True,
+        skips=(
+            # FIXME: incorrectly tries to pass a rhs scalar
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestJit",
+                "test_jit_alias_remapping",
+            ),
+            # TensorIterator does not support symbolic shapes.
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestEagerFusionOpInfo",
+                "test_aot_autograd_symbolic_exhaustive",
+            ),
+        ),
+    ),
+    BinaryUfuncInfo(
+        "igammac",
+        aliases=("torch.special.gammaincc",),
+        dtypes=floating_types_and(torch.bfloat16, torch.float16),
+        dtypesIfCUDA=floating_types(),
+        lhs_make_tensor_kwargs=dict(low=0, high=10),
+        rhs_make_tensor_kwargs=dict(low=0, high=20),
+        supports_rhs_python_scalar=False,
+        supports_inplace_autograd=False,
+        supports_gradgrad=False,
+        supports_forward_ad=True,
+        skips=(
+            # FIXME: incorrectly tries to pass a rhs scalar
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestJit",
+                "test_jit_alias_remapping",
+            ),
+            # TensorIterator does not support symbolic shapes.
```

Comment:
It seems weird to see this test skipped for this reason when other ops that use TensorIterator don't have this same skip. 
It feels like we shouldn't have to do much special things to get symbolic shapes support  because this is just a plain old binary op.

Reply:
I can help take a look if you have a paste.

Reply:
Okay I can see this from the new test failures now:
```
2024-08-13T10:55:47.6707061Z RuntimeError: TensorIterator does not support symbolic shapes; please implement this operator in torch/_refs using the elementwise or reduction helpers (look at backtrace to find out what operator this is)
```
It looks like the fix involves adding a "reference" function. If you are up for fixing these as well it should be fairly straightforward - just a copy-paste of one of the other element-wise examples.